### DISCUSSION TITLE
How to restore the code from an app I accidentally deleted?

### DISCUSSION THREAD

**Discussion by ngamup** [Authenticator-Extension/Authenticator#1329] [Q&A] [answered]
Hello fellows, 
I want to ask if I can get back the code I added for my Facebook 2FA to the Authenticator App. Without this code, I can't log in to my Facebook account as it keeps asking me for the 2FA code to sign in without giving me any other options to log in. Any ideas?  
![z5965939797872_be9b5acc2d92a4a69abdda158fa0f2c8](https://github.com/user-attachments/assets/45d86486-4ecf-4ef0-85aa-31ecaba0bdea)

**Comment by mymindstorm** [comment, marked as answer]
If you lost your codes and do not have a backup, there is nothing we can do to recover them. You can find guidance to recover accounts with lost codes [here](https://authenticator.cc/docs/en/lost-codes).
